sample-kotlin/src/main/java/com/infullmobile/android/infullmvp/sample/sample_mvp_card/SampleMvpCardPresenter.kt

```diff
@@ -0,0 +1,15 @@
+package com.infullmobile.android.infullmvp.sample.sample_mvp_card
+
+import android.net.Uri
+import android.os.Bundle
+import com.infullmobile.android.infullmvp.Presenter
+
+open class SampleMvpCardPresenter(
+        view: SampleMvpCardView,
+        private val model: SampleMvpCardModel
```

Comment:
shouldn't we be injecting the navigation here ? Just for the sake of presenting how to use dependecies from the upper layer.